integrated_channels/integrated_channel/models.py

```diff
@@ -105,6 +105,16 @@ def transmit_content_metadata(self, user):
         transmitter = self.get_content_metadata_transmitter()
         transmitter.transmit(exporter.export())
 
+    def unlink_inactive_learners(self):
+        """
+        Iterate over each learner and unlink inactive learners.
+
+        This method iterates over each enterprise learner and unlink learner
+        from the enterprise if the learner is marked inactive in the related
+        integrated channel.
+        """
+        raise NotImplementedError('Implemented in concrete subclass.')
```

Comment:
Why is it required for each child classes to implement this ? reason ?

Reply:
This is not required, we put it in base configuration model as in future we might want to extend this functionality for other types of integrated channels other than SAP.